src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -380,6 +382,9 @@ def voronoi_cells(V):
 
     - ``V`` -- a corrected Voronoi diagram
 
+    - ``vertical_lines`` -- list (default: ``()``); indices of the
+      vertical lines
```

Comment:
```suggestion
    - ``vertical_lines`` -- tuple (default: ``()``); indices of the
      vertical lines
```
A list input with a tuple as the default is strange.